internal/services/storage/storage_account_resource.go

```diff
@@ -1261,6 +1105,178 @@ func resourceStorageAccount() *pluginsdk.Resource {
+		}
+
+		resource.Schema["primary_queue_host"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeString,
+			Computed:   true,
+			Deprecated: "`primary_queue_host` will be moved to the `azurerm_storage_account_queue_properties` resource in v4.0 of the provider.",
+		}
+
+		resource.Schema["primary_queue_microsoft_endpoint"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeString,
+			Computed:   true,
+			Deprecated: "`primary_queue_microsoft_endpoint` will be moved to the `azurerm_storage_account_queue_properties` resource in v4.0 of the provider.",
+		}
+
+		resource.Schema["primary_queue_microsoft_host"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeString,
+			Computed:   true,
+			Deprecated: "`primary_queue_microsoft_host` will be moved to the `azurerm_storage_account_queue_properties` resource in v4.0 of the provider.",
+		}
+
+		resource.Schema["secondary_queue_endpoint"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeString,
+			Computed:   true,
+			Deprecated: "`secondary_queue_endpoint` will be moved to the `azurerm_storage_account_queue_properties` resource in v4.0 of the provider.",
+		}
+
+		resource.Schema["secondary_queue_host"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeString,
+			Computed:   true,
+			Deprecated: "`secondary_queue_host` will be moved to the `azurerm_storage_account_queue_properties` resource in v4.0 of the provider.",
+		}
+
+		resource.Schema["secondary_queue_microsoft_endpoint"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeString,
+			Computed:   true,
+			Deprecated: "`secondary_queue_microsoft_endpoint` will be moved to the `azurerm_storage_account_queue_properties` resource in v4.0 of the provider.",
+		}
+
+		resource.Schema["secondary_queue_microsoft_host"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeString,
+			Computed:   true,
+			Deprecated: "`secondary_queue_microsoft_host` will be moved to the `azurerm_storage_account_queue_properties` resource in v4.0 of the provider.",
+		}
+
+		// lintignore:XS003
+		resource.Schema["static_website"] = &pluginsdk.Schema{
+			Type:       pluginsdk.TypeList,
+			Optional:   true,
+			Deprecated: "As the `static_website` field requires reaching out to the dataplane, to better support private endpoints and storage accounts with public network access disabled, new storage accounts will be required to use the `azurerm_storage_account_static_website` resource instead of the exposed `static_website` field in the storage account resource itself.",
+			MaxItems:   1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"error_404_document": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
+					},
+					"index_document": {
+						Type:         pluginsdk.TypeString,
+						Optional:     true,
+						ValidateFunc: validation.StringIsNotEmpty,
+					},
+				},
+			},
```

Comment:
I think you have an erroneous `,` here... 
```suggestion
			}
```